plugins/Delay/DelayControlsDialog.cpp

```diff
@@ -22,10 +22,13 @@
  *
  */
 
-#include "delaycontrolsdialog.h"
-#include "delaycontrols.h"
+#include "DelayControlsDialog.h"
+#include "DelayControls.h"
 #include "embed.h"
 #include "TempoSyncKnob.h"
+#include "../Eq/EqFader.h"
```

Comment:
looks like i replied to this twice yesterday. with out clicking the Comment button. I also forgot to mention that the model connected to it, is dB not a multiplier, so the fader acts in a logarithmic  motion. This was requested by dizzy in #1528.



What i was trying to say is that this fader is just a wrapper around the current fx mixer fader, it was not designed to replace the current fader. Im not sure if this should go in the main code, possibly in Fader.h, or if i should just copy the class to the delay plugin as it is not used in the core of lmms.